website/content/docs/getting-started.md

```diff
@@ -156,9 +156,9 @@ $ heron-tracker
 ... Using config file: $HOME/.herontools/conf/heron_tracker.yaml
 ```
 
-You can reach Heron Tracker in your browser at [http://localhost:8888](http://localhost:8888)
-and see something like the following upon successful submission of the topology: 
-![alt tag](http://twitter.github.io/heron/img/heron-tracker.png)
+You can reach Heron Tracker in your browser at ``http://localhost:8888``
```

Comment:
how come you removed the localhost link here but keep it below.. it's convenient to be able to click that when getting started.